src/app/menu/menu.component.css

```diff
@@ -1,69 +1,28 @@
-.menu{
+.example-container {
```

Comment:
i think that bad class name, rename .example to menu-container
and do it everywhere in your code